src/content/developers/docs/blocks/index.md

```diff
@@ -19,54 +19,68 @@ To ensure that all participants on the Ethereum network maintain a synchronized
 
-## Proof-of-work protocol {#proof-of-work-protocol}
+- Validating nodes have to stake at least 32 ETH into a deposit contract as collateral against bad behaviour. This helps protect the network because provably dishonet activity leads to some or all of that stake being destroyed.
+- In every slot (spaced 6 seconds apart) a validator is randomly selected to be the block proposer. They bundle transactions together, execute them and determine a new 'state'. They wrap this information int a block and pass it around to other validators.
+- Other validators who hear about a new block re-execute the transactions to ensure they agree with the proposed change to the global state. Assuming the block is valid they add it to their own database.
+- If a validator hears about two conflicting blocks for the same slot they use their fork-choice algorithm to pick the one supported by the most staked ETH.
 
-Proof-of-work means the following:
-
-- Mining nodes have to spend a variable but substantial amount of energy, time, and computational power to produce a “certificate of legitimacy” for a block they propose to the network. This helps protect the network from spam/denial-of-service attacks, among other things, since certificates are expensive to produce.
-- Other miners who hear about a new block with a valid certificate of legitimacy must accept the new block as the canonical next block on the blockchain.
-- The exact amount of time needed for any given miner to produce this certificate is a random variable with high variance. This ensures that it is unlikely that two miners produce validations for a proposed next block simultaneously; when a miner produces and propagates a certified new block, they can be almost certain that the block will be accepted by the network as the canonical next block on the blockchain, without conflict (though there is a protocol for dealing with conflicts as well in the case that two chains of certified blocks are produced almost simultaneously).
-
-[More on mining](/developers/docs/consensus-mechanisms/pow/mining/)
+[More on proof-of-stake](/developers/docs/consensus-mechanisms/pos)
 
 ## What's in a block? {#block-anatomy}
 
-- `timestamp` – the time when the block was mined.
-- `blockNumber` – the length of the blockchain in blocks.
-- `baseFeePerGas` - the minimum fee per gas required for a transaction to be included in the block.
-- `difficulty` – the effort required to mine the block.
-- `mixHash` – a unique identifier for that block.
-- `parentHash` – the unique identifier for the block that came before (this is how blocks are linked in a chain).
-- `transactions` – the transactions included in the block.
-- `stateRoot` – the entire state of the system: account balances, contract storage, contract code and account nonces are inside.
-- `nonce` – a hash that, when combined with the mixHash, proves that the block has gone through [proof-of-work](/developers/docs/consensus-mechanisms/pow/).
```

Comment:
I think we should also show what's contained in the execution side of the block below the consensus side, of course taking out what is no longer relevant, perhaps making note of these changes. Would be helpful for developers.

Reply:
done!